dataeng/jobs/createJobs.groovy

```diff
@@ -116,6 +117,7 @@ def taskMap = [
     PROGRAM_ENROLLMENT_REPORTS_JOB: ProgramEnrollmentReportsJob,
     READ_REPLICA_EXPORT_TO_S3_JOB: ReadReplicaExportToS3Job,
     SINGLE_SQL_SCRIPT_JOB: SingleSqlScriptJob,
+    SNOWFLAKE_DEMOGRAPHICS_CLEANUP: SnowflakeDemographicsCleanupJob,
```

Comment:
Some additions for this file:    
* This needs to be renamed to SNOWFLAKE_DEMOGRAPHICS_CLEANUP_JOB, in order to match the name of the file in analytics-secure. 
* I'd also suggest adding it to the listView('Tools') list of jobs.